pkg/components/bindings/output_pluggable.go

```diff
@@ -30,7 +30,7 @@ type grpcOutputBinding struct {
 }
 
 // Init initializes the grpc outputbinding passing out the metadata to the grpc component.
-func (b *grpcOutputBinding) Init(metadata bindings.Metadata) error {
+func (b *grpcOutputBinding) Init(ctx context.Context, metadata bindings.Metadata) error {
```

Comment:
replace the ss.Context since you're receiving as a parameter

Reply:
See above